.gitignore

```diff
@@ -0,0 +1,3 @@
+.vscode/
+vppci.py
```

Comment:
W00t, what is this file?

Reply:
vppci.py ? - my frontent to all commands (scripts) in setup.py

Reply:
does it do more or less the same as `python -m ppci`? In other words the file `ppci/__main__.py`?

Reply:
:1st_place_medal: :)) the same